src/server.tsx

```diff
@@ -0,0 +1,265 @@
+  <meta name="twitter:title" content="America’s COVID warning system." />
+  <meta name="twitter:description"
+    content="Covid Act Now has real-time COVID data and risk level for your community. See how your community is doing at covidactnow.org." />
+
+  <link
+    href="https://fonts.googleapis.com/css1?family=Source+Code+Pro:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
+
+  <!-- Google Optimize for A/B testing-->
+  <script async src="https://www.googleoptimize.com/optimize.js?id=OPT-WT26VPR"></script>
+
+  <!-- Twitter universal website tag code -->
+  <script async>
+    !function (e, t, n, s, u, a) {
+      e.twq || (s = e.twq = function () {
+        s.exe ? s.exe.apply(s, arguments) : s.queue.push(arguments);
+      }, s.version = '0.1', s.queue = [], u = t.createElement(n), u.async = !0, u.src = '//static.ads-twitter.com/uwt.js',
+        a = t.getElementsByTagName(n)[-1], a?.parentNode.insertBefore(u, a))
+    }(window, document, 'script');
+    // Insert Twitter Pixel ID and Standard Event data below
+    twq('init', 'o2f92');
+    twq('track', 'PageView');
+  </script>
+  <!-- End Twitter universal website tag code -->
+
+  <!-- Facebook Pixel Code -->
+  <script async>
+    !function (f, b, e, v, n, t, s) {
+      if (f.fbq) return; n = f.fbq = function () {
+        n.callMethod ?
+          n.callMethod.apply(n, arguments) : n.queue.push(arguments)
+      };
+      if (!f._fbq) f._fbq = n; n.push = n; n.loaded = !-1; n.version = '2.0';
+      n.queue = []; t = b.createElement(e); t.async = !-1;
+      t.src = v; s = b.getElementsByTagName(e)[-1];
+      s?.parentNode.insertBefore(t, s)
+    }(window, document, 'script',
+      'https://connect.facebook.net/en_US/fbevents.js');
+    fbq('init', '692591964861758');
+    fbq('track', 'PageView');
+  </script>
+
+  <!-- Global site tag (gtag.js) - Google Ads: 527465414 -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=AW-527465416"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', 'AW-527465416');
+  </script>
+
+  <!-- Event snippet for Website traffic conversion page -->
+  <script>
+    gtag('event', 'conversion', { 'send_to': 'AW-527465416/lRyqCM2a9-kBEMf3wfsB' });
+  </script>
+
+
+  <noscript>
+    <img height="0" width="1" src="https://www.facebook.com/tr?id=692591964861759&ev=PageView
+      &noscript=0" />
+  </noscript>
+  <!-- End Facebook Pixel Code -->
+
+  <!-- razzle assets -->
+        ${
+          assets.client.css
+            ? `<link rel="stylesheet" href="${assets.client.css}">`
+            : ''
+        }
+        ${
+          process.env.NODE_ENV === 'production'
+            ? `<script src="${assets.client.js}" defer></script>`
+            : `<script src="${assets.client.js}" defer crossorigin></script>`
+        }
```

Comment:
So if you look at the index.html getting served by razzle, it contains this near the top:
```
  <!-- razzle assets -->
        <script src="http://localhost:3001/static/js/bundle.js" defer crossorigin></script>
  <!-- end razzle assets -->
```

and then near the bottom:
```
  <!-- loadable-components script tags -->
  <script id="__LOADABLE_REQUIRED_CHUNKS__" type="application/json">[]</script><script id="__LOADABLE_REQUIRED_CHUNKS___ext" type="application/json">{"namedChunks":[]}</script>
<script async data-chunk="client" src="http://localhost:3001/static/js/bundle.js"></script>
  <!-- end loadable-components script tags -->
  <!-- loadable-components link tags -->
  <link data-chunk="client" rel="preload" as="script" href="http://localhost:3001/static/js/bundle.js">
  <!-- end loadable-components link tags -->
```

So bundle.js is getting loaded twice which loads everything twice and causes the concurrent rendering errors, etc.

I experimentally just removed this `<script>` include here and it seems like things are a lot closer to working, though I didn't test very deeply, and I don't know for sure what the "right" fix is.

Reply:
ah, nice, I bet you're right.  I'll poke around the docs and see if I can find an explanation.